### ISSUE TITLE
Check absolute path description of command

### ISSUE THREAD

**Issue by kenchan0130** [koalaman/shellcheck#1410]
#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

If we use shell with cron etc, we describe the command in absolute path unless we set PATH.

```sh
#!/bin/sh

extractingTxt=$(< /path/to/file /usr/bin/grep extracting_txt)

/bin/echo "$extractingTxt"

exit 0
```

Therefore, we often want to enforce absolute path.

```sh
#!/bin/sh

extractingTxt=$(< /path/to/file /grep extracting_txt)

echo "$extractingTxt"

exit 0
```

#### Here's what shellcheck currently says:

It does not say anything.

#### Here's what I wanted or expected to see:

I expect the it to say "the `grep` is not absolute path and the `echo` is not absolute path".

**Comment by koalaman** [comment]
I can see your argument for some cases, but I think this is better fixed by setting the PATH in crontab correctly, and generally letting the user be responsible for correctly configuring their environment.

There are downsides to hard-coding all paths all the time: The user can no longer use `brew` or `cabal` to install their own versions of utilities. Different OS put utilities in different places, such as `/bin/bzip2` on Ubuntu and `/usr/bin/bzip2` on FreeBSD. The shell will also be unable to use its builtin versions of commands like `echo` or `[`.
